x-pack/legacy/plugins/security/public/views/management/roles_grid/components/roles_grid_page.tsx

```diff
@@ -191,29 +191,22 @@ class RolesGridPageUI extends Component<Props, State> {
         },
       },
       {
-        field: 'metadata._reserved',
-        name: (
-          <EuiToolTip content={reservedRoleDesc}>
```

Comment:
Removed tooltip to be consistent with the User grid page. The tooltip ended up swallowing the click event, which prevents users from initiating a sort on this column.